### ISSUE TITLE
Sticky note out of sync. Error message, Lost of Ink 

### ISSUE THREAD

**Issue by agorog** [IBVSoftware/CollaBoard-Feedback#66]
Projekt Code: 1188
Projekt Schlüssel: 1lr8i
ibv-collaboard:open?code=1188&key=1lr8i

Date: 28th of January
Time: 09:19
Device: Surface Hub and on Surface Pro 4
Version: 1.2.9.0

see uploaded document 

[syncProblem.docx](https://github.com/IBVSoftware/CollaBoard-Feedback/files/736788/syncProblem.docx)

**Comment by gsantopaolo** [comment]
please define what is a ghost tile for you, and what is your meaning of out of sync because i cannot understand what the problem is

**Comment by gsantopaolo** [comment]
what is the meaning of this? 
![image](https://cloud.githubusercontent.com/assets/9937149/22396352/9798a1b8-e557-11e6-8acb-954f2506af2f.png)

**Comment by gsantopaolo** [comment]
sorry this is too complicated.
I see there that you are trying to describe more than one issue but it becomes very hard to follow and understand...

**Comment by agorog** [comment]
I tried to make it simpler, not easy. You work on two devices on same project. Have some sticky notes and a grid library move sticky notes in/out library and ink. I had several error message, so project closed. Project was 1424 / iynxh. 28.01.2017 13:30 - 13.50. There are situation where you completly loose a tile, where a tile is displayed twice. As soon as something goes wrong with a tile and then you try to work on this tile on the other device you get the error. Example for a ghost tile: 
![grafik](https://cloud.githubusercontent.com/assets/19590642/22396724/cddcf0a4-e560-11e6-9ae0-2f0b4951d507.png)

**Comment by agorog** [comment]
Then moving an inking the tile with the 3 and you still see the ghost tile without inking 
![grafik](https://cloud.githubusercontent.com/assets/19590642/22396730/074b6ea6-e561-11e6-8dc1-5ec1ec1ea012.png)

**Comment by agorog** [comment]
Then I move an ink the ghost tile (3 without black line). 
![grafik](https://cloud.githubusercontent.com/assets/19590642/22396734/360a7aca-e561-11e6-9513-12c10a2a3cf2.png)

**Comment by agorog** [comment]
and have no 3 tiles

**Comment by agorog** [comment]
Closing / reopening shows project with the latest tile, i.e. origin with black line disappeared and we lost black line. 
![grafik](https://cloud.githubusercontent.com/assets/19590642/22396752/92a6b3ac-e561-11e6-9fdf-a230ca310781.png)

**Comment by agorog** [comment]
I didn't experience this any more with 1.3.0.0

**Comment by gsantopaolo** [comment]
ok thx closing
